cmd/ecloud/ecloud_loadbalancernetwork.go

```diff
@@ -0,0 +1,236 @@
+package ecloud
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/ukfast/cli/internal/pkg/factory"
+	"github.com/ukfast/cli/internal/pkg/helper"
+	"github.com/ukfast/cli/internal/pkg/output"
+	"github.com/ukfast/sdk-go/pkg/service/ecloud"
+)
+
+func ecloudLoadBalancerNetworkRootCmd(f factory.ClientFactory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "loadbalancernetwork",
+		Short: "sub-commands relating to load balancer networks",
+	}
+
+	// Child commands
+	cmd.AddCommand(ecloudLoadBalancerNetworkListCmd(f))
+	cmd.AddCommand(ecloudLoadBalancerNetworkShowCmd(f))
+	cmd.AddCommand(ecloudLoadBalancerNetworkCreateCmd(f))
+	cmd.AddCommand(ecloudLoadBalancerNetworkUpdateCmd(f))
+	cmd.AddCommand(ecloudLoadBalancerNetworkDeleteCmd(f))
+
+	return cmd
+}
+
+func ecloudLoadBalancerNetworkListCmd(f factory.ClientFactory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "list",
+		Short:   "Lists load balancer networks",
+		Long:    "This command lists load balancer networks",
+		Example: "ukfast ecloud loadbalancer list",
```

Comment:
I think the example given here should be `ukfast ecloud loadbalancernetwork list`

Reply:
Ahh good spot, sorted